samples/nrf9160/cloud_simple/README.rst

```diff
@@ -0,0 +1,110 @@
+.. _cloud_simple:
+
+nRF9160: Cloud Simple
+#####################
+
+Sample that connects to and communicates with a compatible cloud service using the respective firmware cloud backend. The sample connects via cellular network (LTE) and publishes a custom string in intervals or upon a button trigger, to the cloud service.
+
+Overview
+********
+
+The ``cloud_simple`` sample demonstrates how to use the generic cloud API ``subsys/net/lib/cloud`` to interface with multiple cloud backends. The current version of the ``cloud_simple`` sample supports the following libraries as cloud backends:
+
+ -  :ref:`lib_nrf_cloud`
+ -  :ref:`lib_aws_iot`
+
+To swap between cloud backends simply change the option ``CONFIG_CLOUD_BACKEND`` to match the configuration string of a compatible cloud backend. The identifying string for the different cloud backends are listed below.
+
+ - **NRF Cloud**, configuration string ->"NRF_CLOUD"
+ - **AWS IoT**, configuration string -> "AWS_IOT"
+
+Setup
+*****
+
+For setting up the different cloud backends please see the corresponding documentations. Each cloud backend has spesific setup steps that must be executed before it can be used. These setup steps cover generating and flashing *certificates* used for the cloud connection, setting up the correct connection *endpoint* etc..
```

Comment:
````Suggestion
For setting up the different cloud backends, refer the documentation on :ref:`lib_nrf_cloud` and :ref:`lib_aws_iot` . 
Each cloud backend has specific setup steps that must be executed before it can be used.
These setup steps describes the following:
#. generating and flashing *certificates* used for the cloud connection
#. setting up the correct connection *endpoint* 
````